.opencode/command/sync_with_main_and_resolve_conflicts.md

```
@@ -0,0 +1,130 @@
+6. Verification must run before declaring success.
+7. Push is normal `git push` only; never force-push.
+</workflow_contract>
+
+<userMessage>
+$ARGUMENTS
+</userMessage>
+
+<process>
+
+<step_1>
+## Step 1: Establish context + todo list
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)